### DISCUSSION TITLE
Explicit labels in the transition matrix

### DISCUSSION THREAD

**Discussion by i-m-amit** [sepal-contrib/sdg_15.3.1#82] [Ideas]
As the transition matrix is not a familiar tool to asses land  cover change, a user might not understand the matrix properly as we experienced during the last training. Therefore, I would consider changing the transition labels from symbols to explicit text and some explanation before the matrix. What do you think?

**Comment by 12rambau** [comment]
I think symbol will also loose people, I'm working on the reclassify panel to help people use custom/national land cover maps. I will create an  "advance parameter" `ExpansionPanel`that will include the transition matrix and the 2 custom `AssetSelect`. 

In the meantime i'll work on #60 to give complete information to the end user.

I'm moving forward in https://github.com/12rambau/sdg_indicators_module/tree/reclassify

**Comment by i-m-amit** [comment]
This is what I meant.
We could also add some explanation like rows contain the initial land cover, column contain the target land cover etc.
<img width="1108" alt="Screenshot 2021-10-26 at 1 56 30 PM" src="https://user-images.githubusercontent.com/12950330/138840347-3041ec4a-78d2-436e-8bc3-ccdac289f13f.png">

**Reply by 12rambau** [reply]
ahhh, then it's super easy, I'll add the description of rows and column in the doc. I want to force people to read the manual at least once to understand the application, it will lighten what we write in the GUI and allows us to reference it when people have questions

**Comment by 12rambau** [comment]
I opened an issue (https://github.com/12rambau/sdg_indicators_module/issues/85) for this matter, I aggree it needs to be handled
